app/src/main/java/com/github/swent/echo/compose/components/EventInfoSheet.kt

```diff
@@ -88,11 +94,12 @@ fun EventInfoSheet(
             )
             Text(
                 modifier = Modifier.padding(top = 24.dp),
-                text = event.organizer!!.name,
+                text = event.organizer?.name ?: event.creator.name,
                 style =
                     TextStyle(
                         fontSize = 20.sp,
                         fontWeight = FontWeight(600),
+                        color = MaterialTheme.colorScheme.secondary
```

Comment:
Instead of defining a custom text style, we could use the one defined by `MaterialTheme`, e.g. `style = MaterialTheme.typography.titleMedium.copy(color = MaterialTheme.colorScheme.secondary)`.